Update navigation to add link to security workshop

webhamster opened the pull request.

webhamster commented:
Without this patch, there is no link to the security workshops page on oauth.net.

aaronpk commented:
Sorry, got pulled away yesterday with other work. I don't think this is an appropriate top-level nav item, I'd rather it be more broad. So instead, I've added a new page "Events" to the top nav, where these workshops are included in the list of events along with some others. I've also added a link to this page on the "Security" page. Thanks!

https://oauth.net/events/

webhamster commented:
That is a great solution! I like that list!

Am 22.11.18 um 18:57 schrieb Aaron Parecki:
>
> Sorry, got pulled away yesterday with other work. I don't think this
> is an appropriate top-level nav item, I'd rather it be more broad. So
> instead, I've added a new page "Events" to the top nav, where these
> workshops are included in the list of events along with some others.
> I've also added a link to this page on the "Security" page. Thanks!
>
> https://oauth.net/events/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aaronpk/oauth.net/pull/187#issuecomment-441095721>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFdhO4cwNjhVlahqE1yfwsVISez9UWcks5uxuWcgaJpZM4YrIU9>.
>
